farmdata2_modules/cypress/downloads/tableDownload.csv

```diff
@@ -0,0 +1,3 @@
+cool,works?,hello,
```

Comment:
Looks like there is an extra comma at the end of the line.  You'll want to be sure that isn't there on the headers or the data rows in the final verson.